### ISSUE TITLE
"Open Files" panel loses close "x" with long filenames

### ISSUE THREAD

**Issue by we3geeks** [Komodo/KomodoEdit#168]
The positioning of the "x" icon to close files in the "Open Files" panel is based, at least in part, on the length of the longest filename open in the editor. Unfortunately, this causes that "x" icon to not be visible if a file with a long enough name is opened.

Compare these two screenshots: the only difference is the width of the "Open Files" panel... one shows the close "x" icon, the other does not.

![screen shot 2015-04-13 at 3 21 41 pm 3](https://cloud.githubusercontent.com/assets/2390335/7126356/ce733ef8-e1f5-11e4-8939-ec5d01177bd8.png)

![screen shot 2015-04-13 at 3 22 02 pm 3](https://cloud.githubusercontent.com/assets/2390335/7126369/e185dafa-e1f5-11e4-8ced-d5e28931dad9.png)

The close "x" icon should be positioned based solely on the width of the "Open Files" panel so that it is always available.

Build info: Komodo IDE, version 9.0.0, build 87165, platform macosx. Built on Wed Mar 18 21:57:11 2015.

**Comment by Naatan** [comment]
Yup you're totally right, unfortunately XUL is a PITA with this type of stuff. Will dig in again when time allows.
